### DISCUSSION TITLE
Cheaper/free "low tier" seats in organization

### DISCUSSION THREAD

**Discussion by OliverRC** [community/community#73571] [Enterprise]
We are a small company (relatively speaking) of around ~50 people. 
However, only 3 of the 50 are developers.

We use GitHub to maintain and run the development side of the business and, therefore, are currently on the Enterprise pricing plan, which is ~$20 per/seat. We are happy with the value for developers.

However, I would love to allow the greater company to view issues, potentially even PRs or the code, and even allow them to log a bug.
That said, $20 simply does not make sense from a value standpoint. These users will barely use 90% of the features of a full seat.

We'd love it if there were a way to give users access to a low-tier seat plan in our organization.
They would have minimal functionality. However, they could see how development items are going.

One idea could be a user with access to only the project management features. 
That way, they can still comment, create issues, etc, but would not need to have the complete developer set of features and, therefore, be a much more affordable (if not free) addition.

In Azure develops this feature is available as "Stakeholder" access.
https://learn.microsoft.com/en-us/azure/devops/organizations/security/stakeholder-access?view=azure-devops

**Comment by moonwitch** [comment]
This would indeed be very interesting. You could add them as outside collaborators, but that still uses a seat. Even a machine user uses a seat.

**Reply by OliverRC** [reply]
I think the idea here is not to spend $20 x 47 = ~$940 to allow members of our company to access Github infrequently and use some tiny fraction of the features. 

Only 3 out of 50 users require the full enterprise seat functionality.

**Comment by TaylorRichberger** [comment]
We're in a similar situation.  It doesn't make much sense that an organization of 3 developers working and 27 other users who download packages and open issues would cost the same amount as 30 full-time developers using all the features available to them.

**Comment by ryanmelt** [comment]
Hello,

I would love to upgrade my company to Github Enterprise, but can't because you charge the same per month for users with write access, vs those with just read only access.

All of our customers are given read only access to our private repos, which we can do for free with the Free plan.

But I can't afford that on the Enterprise plan. I would happily pay for the Enterprise plan if we could continue to add unlimited read only collaborators for free, but pay for those with write access.

Please consider this change to your business model and forward on to whichever executive could make that happen.

Thanks,
Ryan Melton,
CEO - OpenC3, Inc.
